server/chat/backend/agent/tools/bitbucket/apply_fix_tool.py

```
@@ -0,0 +1,189 @@
+
+def _parse_repository(repo_string: str) -> tuple[Optional[str], Optional[str]]:
+    """Parse 'workspace/repo_slug' string into tuple."""
+    if not repo_string:
+        return None, None
+    parts = repo_string.split("/")
+    if len(parts) == 2 and parts[0] and parts[1]:
+        return parts[0], parts[1]
+    return None, None
+
+
+def bitbucket_apply_fix(
```

Ruff (0.15.15)

[warning] Too many return statements (14 > 6)

(PLR0911)